repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/servlet/http/HttpServletResponseWrapper.scala

```diff
@@ -223,6 +225,10 @@ class HttpServletResponseWrapper(originalResponse: HttpServletResponse, headerMo
     addHeader(name, DateUtils.formatDate(new Date(timeSinceEpoch)))
 
   override def addHeader(name: String, value: String): Unit = {
```

Comment:
Should would add the warning to `setContentLength`, `setContentType`, & `setCharacterEncoding`?
Currently we check and just silently do nothing if it is committed.